packages/renderer/src/lib/ui/NavPage.svelte

```diff
@@ -40,13 +40,13 @@ export let searchEnabled = true;
               class="w-full py-2 outline-none text-sm bg-charcoal-800 rounded-sm text-gray-700 placeholder-gray-700" />
           </div>
         </div>
-        <div class="flex flex-1 px-5">
+        <div class="flex flex-1 px-5" role="group" aria-label="bottomAdditionalActions">
```

Comment:
is it more a `navigation` pattern ?

Reply:
I would say not really:
Navigation: `The navigation role is used to identify major groups of links used for navigating through a website or page content.`
whilst Group: `The group role identifies a set of user interface objects that is not intended to be included in a page summary or table of contents by assistive technologies.`

Reply:
Group fits better here I would say as we are talking about some additional actions that can be done, not navigating through the document/page, wdyt?